packages/apollo-server-lambda/src/lambdaApollo.ts

```diff
@@ -30,7 +30,7 @@ export function graphqlLambda(
     event,
     context,
     callback,
-  ): void => {
+  ): any => {
```

Comment:
I had to change this to any to comply with the changes. If you have a better approach, please suggest and I will update it.